go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/gorilla/mux v1.8.1
 	github.com/gorilla/securecookie v1.1.2
-	github.com/gorilla/websocket v1.5.1
+	github.com/gorilla/websocket v1.5.3
```

Comment:
Please can you limit the go mod update to just the ones relevant for this PR. Ta

Reply:
We dont want to bump this one just yet.

Reply:
@tomponline Done!